pages/common/git-tag.md

```diff
@@ -0,0 +1,19 @@
+# git tag
+
+> Create, list, delete or verify tags
+
+- List all tags
+
+`git tag`
+
+- Create a named tag pointing to the current commit
```

Comment:
I think "named tag" is redundant and may create unneeded confusion. Are there "unnamed tags" in git?